mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -340,6 +340,55 @@ function Row(props: {
   );
 }
 
+function extractPath(strings: string[]): string {
+  let returnString = "";
```

Comment:
We have a linting rule to use single quotes

Reply:
Interesting. I do not have linter rules on my editor, neither there is not test doing linting. Is there a specific command to run linter?

Reply:
I'm not sure, I use a plugin so I see the errors in real time. That would be better, because if you run linter you'll just get warnings for all the files and all the errors that aren't your doing :D So if you can, get a plugin that utilizes the linting config and just try and keep your own code clean.

Reply:
Actually it might not even be a plugin, it's maybe just included in the IDE (I use IntelliJ), but depending on what IDE you use, you can try and search for settings or tools to help automatically find and fix formatting errors.